utils/training_loop.py

```diff
@@ -1,270 +1,104 @@
-from utils.NN_utils import *
-from utils.airsim_manager import AirsimManager
-from utils.logger import Logger
-from utils.rl import RL
-
-
-def training_loop_ido(config):
-    # classes init
-    logger = Logger(config)
-    airsim = AirsimManager(config)
-    nn_handler = NN_handler(config)
-    rl = RL(config, logger, airsim, nn_handler)
+import matplotlib.pyplot as plt
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import DummyVecEnv
+from gym_enviroment import AirSimGymEnv
+from airsim_manager import AirsimManager
+from stable_baselines3.common.logger import configure
+import pandas as pd
+from config import Config
+import random
+
+def training_loop_ariel_step(config, path):
+    all_rewards = []
+    new_logger = configure(path, ["stdout", "csv", "tensorboard"])
+    env = DummyVecEnv([lambda: AirSimGymEnv(config, AirsimManager(config))])
+    model = PPO('MlpPolicy', env, verbose=1, learning_rate=0.0001, n_steps=160, batch_size=160)
```

Comment:
consider moving the variables to config or when defining an experiment

Reply:
fixed